package-lock.json

```diff
@@ -5161,6 +5166,12 @@
           "resolved": "https://registry.npmjs.org/regexp-tree/-/regexp-tree-0.1.13.tgz",
           "integrity": "sha512-hwdV/GQY5F8ReLZWO+W1SRoN5YfpOKY6852+tBFcma72DKBIcHjPRIlIvQN35bCOljuAfP2G2iB0FC/w236mUw==",
           "dev": true
+        },
+        "semver": {
```

Comment:
This can probably be removed.

Reply:
This was automatically added by running `npm install`, won't removing it break the automated build? This will probably recur when adding a new dependency?

Reply:
If there weren't other intentional changes to `package-lock.json`, I'd agree that we should remove the whole file from the PR, but since those are needed, I think it's fine to leave this in. It'll definitely get added by some other random PR if it isn't added by this one.

`semver` got removed and then re-added from the `package.json` by mistake last week, so I'm not surprised that `npm` hasn't sorted out whether or not it's supposed to be in the lock file yet.